app/src/main/java/org/xwiki/android/sync/activities/SignInViewFlipper.kt

```diff
@@ -183,7 +183,7 @@ class SignInViewFlipper(
         return appCoroutineScope.launch {
             val user = userAccountsRepo.createAccount(
                 UserAccount(
-                    accountName,
+                    "$accountName@temp",
```

Comment:
What is `temp` and why it was added here? just to identify that this user is temporal?

Reply:
Yes, so earlier two accounts are created simultaneously although one of them is intended to be deleted, it wasn't deleted. So by adding temp, we distinguish that this account is temporary and will be deleted now.